### PR TITLE
de_de updated

### PR DESCRIPTION
**Pull request by ichbineinNerd**

### CODE REVIEW

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 1]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
 of.options.shaders.CLOUD_SHADOW=Wolkenschatten
 
 of.options.shaders.OLD_HAND_LIGHT=Alte Handbel.
-of.options.shaders.OLD_HAND_LIGHT.tooltip.1=Alte Handbeleuchtung
+of.options.shaders.OLD_HAND_LIGHT.tooltip.1=Alte Handbelichtung
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Handbeleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 2]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
 of.options.shaders.CLOUD_SHADOW=Wolkenschatten
 
 of.options.shaders.OLD_HAND_LIGHT=Alte Handbel.
-of.options.shaders.OLD_HAND_LIGHT.tooltip.1=Alte Handbeleuchtung
+of.options.shaders.OLD_HAND_LIGHT.tooltip.1=Alte Handbelichtung
 of.options.shaders.OLD_HAND_LIGHT.tooltip.2=  Standard - gesteuert vom Shader-Paket
-of.options.shaders.OLD_HAND_LIGHT.tooltip.3=  An - Alte Handbeleuchtung aktiviert
-of.options.shaders.OLD_HAND_LIGHT.tooltip.4=  Aus - Alte Handbeleuchtung deaktiviert
-of.options.shaders.OLD_HAND_LIGHT.tooltip.5=Die alte Handbeleuchtung sorgt dafür, dass leuchtende
+of.options.shaders.OLD_HAND_LIGHT.tooltip.3=  An - Alte Handbelichtung aktiviert
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Handbeleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 3]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
...
 
 of.options.shaders.OLD_HAND_LIGHT=Alte Handbel.
-of.options.shaders.OLD_HAND_LIGHT.tooltip.1=Alte Handbeleuchtung
+of.options.shaders.OLD_HAND_LIGHT.tooltip.1=Alte Handbelichtung
 of.options.shaders.OLD_HAND_LIGHT.tooltip.2=  Standard - gesteuert vom Shader-Paket
-of.options.shaders.OLD_HAND_LIGHT.tooltip.3=  An - Alte Handbeleuchtung aktiviert
-of.options.shaders.OLD_HAND_LIGHT.tooltip.4=  Aus - Alte Handbeleuchtung deaktiviert
-of.options.shaders.OLD_HAND_LIGHT.tooltip.5=Die alte Handbeleuchtung sorgt dafür, dass leuchtende
+of.options.shaders.OLD_HAND_LIGHT.tooltip.3=  An - Alte Handbelichtung aktiviert
+of.options.shaders.OLD_HAND_LIGHT.tooltip.4=  Aus - Alte Handbelichtung deaktiviert
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Handbeleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 4]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
...
+of.options.shaders.OLD_HAND_LIGHT.tooltip.5=Die alte Handbelichtung sorgt dafür, dass leuchtende
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Handbeleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 5]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
...
 of.options.shaders.OLD_HAND_LIGHT.tooltip.6=Gegenstände bei alten Shader-Paketen sowohl in der
 of.options.shaders.OLD_HAND_LIGHT.tooltip.7=Haupt-, als auch der Nebenhand erkannt werden.
 
 of.options.shaders.OLD_LIGHTING=Alte Bel.
-of.options.shaders.OLD_LIGHTING.tooltip.1=Alte Beleuchtung
+of.options.shaders.OLD_LIGHTING.tooltip.1=Alte Belichtung
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Beleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 6]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
...
 of.options.shaders.OLD_LIGHTING.tooltip.2=  Standard - gesteuert vom Shader-Paket
-of.options.shaders.OLD_LIGHTING.tooltip.3=  An - Alte Beleuchtung aktiviert
-of.options.shaders.OLD_LIGHTING.tooltip.4=  Aus - Alte Beleuchtung deaktiviert
-of.options.shaders.OLD_LIGHTING.tooltip.5=Die alte Beleuchtung steuert die starre "Vanilla"
-of.options.shaders.OLD_LIGHTING.tooltip.6=Beleuchtung der Block-Seiten.
+of.options.shaders.OLD_LIGHTING.tooltip.3=  An - Alte Belichtung aktiviert
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Beleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 7]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
...
+of.options.shaders.OLD_LIGHTING.tooltip.4=  Aus - Alte Belichtung deaktiviert
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Beleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 8]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
...
+of.options.shaders.OLD_LIGHTING.tooltip.5=Die alte Belichtung steuert die starre "Vanilla"
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Beleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 9]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
...
+of.options.shaders.OLD_LIGHTING.tooltip.6=Belichtung der Block-Seiten.
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Beleuchtung: "Belichtung" is not fully correct

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 10]
```diff
@@ -221,30 +222,30 @@ of.options.shaders.HAND_DEPTH_MUL.tooltip.8=die Unschärfe der tragbaren Gegenst
...
 of.options.shaders.OLD_LIGHTING.tooltip.7=Shader-Pakete mit Schatten variierend nach Sonnenstand
-of.options.shaders.OLD_LIGHTING.tooltip.8=stellen meistens eine bessere Beleuchtung zur Verfügung.
+of.options.shaders.OLD_LIGHTING.tooltip.8=stellen meistens eine bessere Belichtung zur Verfügung.
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Beleuchtung: "Belichtung" is not fully correct

**Comment by ichbineinNerd** [comment]
yeah, you're right. just googled it.

**Comment by ichbineinNerd** [comment]
yay

**Pull request merged**